### PR TITLE
doc: Use the conventional comment syntax

### PR DESCRIPTION
**Pull request by Muhammad-Owais-Warsi**
## Description
Changed the comments syntax from `/* */` to `///` in the codebase.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (no code change)
- [ ] Refactor (refactoring production code)
- [ ] Other 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have updated the [documentation](https://github.com/orhun/kmon/blob/master/README.md) and [changelog](https://github.com/orhun/kmon/blob/master/CHANGELOG.md) accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] [Rustfmt](https://github.com/rust-lang/rustfmt) and [Rust-clippy](https://github.com/rust-lang/rust-clippy) passed.

### CODE REVIEW

File: src/util.rs [hunk 1]
```diff
@@ -66,13 +67,13 @@ pub fn exec_cmd(cmd: &str, cmd_args: &[&str]) -> Result<String, String> {
...
- * See <https://ratatui.rs/how-to/develop-apps/panic-hooks/#termion>
- *
- * @return Result
- */
+
+/// Sets up the panic hook for the terminal.
+
+/// See <https://ratatui.rs/how-to/develop-apps/panic-hooks/#termion>
+ 
+/// @return Result
```
Comment (orhun) [inline src/util.rs:75]: Ah, I think we can also get rid of these C-style `@return` and `@param` lines too. Can you remove them as well?

**Review by orhun — COMMENTED** [review]
Thanks for the PR!

File: src/util.rs (hunk 1 above)
Comment (Muhammad-Owais-Warsi) [reply src/util.rs:75]: Do you mean completely removing those lines , if am not wrong

Comment (orhun) [reply src/util.rs:75]: Yes, we can completely remove them.

Comment (Muhammad-Owais-Warsi) [reply src/util.rs]: > Ah, I think we can also get rid of these C-style `@return` and `@param` lines too. Can you remove them as well?

please review and let me know .

File: src/app.rs [hunk 2]
```diff
@@ -104,17 +103,16 @@ pub enum InputMode {
...
 impl InputMode {
-	/**
-	 * Check if input mode is set.
-	 *
-	 * @return bool
-	 */
+	
+	/// Check if input mode is set.
+	 
+
```
[the remark this answers is not in the document]
Comment (Muhammad-Owais-Warsi) [reply src/app.rs]: Just pushed , please check , hope I have not missed anything now 😅

**Comment by orhun** [comment]
CI is failing due to invalid doc comments: https://github.com/orhun/kmon/actions/runs/11220108587/job/31215613896?pr=174

See https://doc.rust-lang.org/reference/comments.html

I guess we are still not ready :)

**Comment by Muhammad-Owais-Warsi** [comment]
> CI is failing due to invalid doc comments: https://github.com/orhun/kmon/actions/runs/11220108587/job/31215613896?pr=174
> 
> See https://doc.rust-lang.org/reference/comments.html
> 
> I guess we are still not ready :)

So sorry, actually it is my first time trying rust. I think I have used `///` inside the functions , but I  think it's not the right way. I'll look into that. Thanks :)

**Comment by orhun** [comment]
No worries :)

**Comment by Muhammad-Owais-Warsi** [comment]
> > CI is failing due to invalid doc comments: https://github.com/orhun/kmon/actions/runs/11220108587/job/31215613896?pr=174
> > See https://doc.rust-lang.org/reference/comments.html
> > I guess we are still not ready :)
> 
> So sorry, actually it is my first time trying rust. I think I have used `///` inside the functions , but I think it's not the right way. I'll look into that. Thanks :)

As per the docs, we have to use `//` for the comments inside the functions. Comments starrting with `///` will be used to document a function specifically.

**Comment by Muhammad-Owais-Warsi** [comment]
> > > CI is failing due to invalid doc comments: https://github.com/orhun/kmon/actions/runs/11220108587/job/31215613896?pr=174
> > > See https://doc.rust-lang.org/reference/comments.html
> > > I guess we are still not ready :)
> > 
> > 
> > So sorry, actually it is my first time trying rust. I think I have used `///` inside the functions , but I think it's not the right way. I'll look into that. Thanks :)
> 
> As per the docs, we have to use `//` for the comments inside the functions.

**Comment by Muhammad-Owais-Warsi** [comment]
On `cargo build` i get this

**Comment by orhun** [comment]
Yup, please. Those look good

**Comment by Muhammad-Owais-Warsi** [comment]
> Yup, please. Those look good

Done 👍

**Comment by orhun** [comment]
Can you run `cargo fmt` and push the changes?

**Comment by Muhammad-Owais-Warsi** [comment]
> Can you run `cargo fmt` and push the changes?



> Can you run `cargo fmt` and push the changes?

Yeah, sure.

**Review by orhun — APPROVED** [review]
Thanks for your contribution (and bearing with me :D)

**Comment by Muhammad-Owais-Warsi** [comment]
> Thanks for your contribution (and bearing with me :D)
Thank you so much , it was great to contribute first time on rust project. You helped me lot. Thanks again

**Pull request merged**